tests/wpunit/TEC/Events/Calendar_Embeds/Calendar_EmbedsTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
```

Comment:
Lets create our own test suite instead. I can take care of that. For now leave it in wpunit and ill move it.